html/pfappserver/lib/pfappserver/PacketFence/Controller/Admin.pm

```diff
@@ -350,6 +352,34 @@ sub configuration :Chained('object') :PathPart('configuration') :Args(0) {
 
 }
 
+=head2 time_offset
+
+time_offset
```

Comment:
That is a name, not a description

Reply:
Agreed, we should have at least one sentence to describe what the method is supposed to do and what the params are.